### PR TITLE
Firefox 54 to be released tomorrow (jun/13)

### PR DESCRIPTION
**Pull request by afmenez**
Firefox 54 will be released tomorrow (june/13).  This patch updates the version table accordingly.

### CODE REVIEW

**Review by chicoxyzzy — APPROVED** [review]
1 comment. otherwise 👍

File: environments.json [hunk 1]
```diff
@@ -601,7 +601,7 @@
     "short": "FF 46",
     "release": "2016-04-26",
     "retire": "2016-06-07",
-    "obsolete": true
+    "obsolete": "very"
```
Comment (chicoxyzzy) [inline environments.json:604]: i think this is a mistake

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]

File: environments.json (hunk 1 above)
Comment (afmenez) [reply environments.json:604]: The rule I am using is true after retirement, and "very" one year later.  This lets me remove one version every time I add a new one.
ESR releases work differently, and usually there is more than one supported version at a time.  That's why I am removing 46 but 45 is still active.

**Comment by afmenez** [comment]
Firefox 54 is already out and this can be merged.

File: environments.json (hunk 1 above)
Comment (chicoxyzzy) [reply environments.json:604]: Oh, 45 is ESR. OK than

**Review by chicoxyzzy — APPROVED** [review]

**Comment by chicoxyzzy** [comment]
Thank you!

**Pull request merged**